docs/Theming.md

```diff
@@ -984,16 +1022,22 @@ const App = () => (
 
 ### Changing the Icon
 
-It is possible to use a completely [custom logout button](./Admin.md#logoutbutton) or you can simply override some properties of the default button. If you want to change the icon, you can use the default `<Logout>` component and pass a different icon as the `icon` prop.
+It is possible to use a completely [custom logout button](./Theming.md#customizing-the-logout-component) or you can simply override some properties of the default button. If you want to change the icon, you can use the default `<Logout>` component and pass a different icon as the `icon` prop.
```

Comment:
This link doesn't exist [custom logout button](./Theming.md#customizing-the-logout-component)
I'm not sure which is the correct one